gcp/testing/flaky.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tools for dealing with flaky tests.
+"""
+from __future__ import absolute_import
+
+
+from flaky import flaky as _flaky
```

Comment:
also confused by what's going on here...I'm sure there's a good reason I just feel bad LGTMing it without know what it is!